source/usage-examples/updateMany.txt

```diff
@@ -3,3 +3,59 @@ Update Multiple Documents
 =========================
 
 .. default-domain:: mongodb
+
+You can update multiple documents in a collection by using the ``UpdateMany()``
+method.
+
+The following example passes a query filter and an update parameter to the
+``UpdateMany()`` method. The query filter matches documents in the
+``listingsAndReviews`` collection if the value of their "address.market"
+field is "Sydney", and the update filter multiplies the ``price`` field
+in the matching documents by a factor of *1.15*:
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /code-snippets/updateMany.go
+   :start-after: begin updatemany
+   :end-before: end updatemany
+   :emphasize-lines: 5
+   :language: go
+   :dedent:
+
+Click here **<TODO: github link to file>** to see a fully runnable example.
+
+Expected Result
+---------------
+
+After you run the code example, the ``price`` field of the matched
+documents will reflect the multiplication. The updated documents will
+appear similar to this one:
+
+.. code-block:: json
+   :copyable: false
+
+   // result truncated
+   { 
+      "_id" : "17438054",
+      ...
+      "name":"Cozy Granny Flat",
+      ...
+      "price": "115.00"
+      ...
+   }
```

Comment:
Question: Since this is an updateMany operation, can we show at least 2 documents updated (the first and last)? A potential solution is to make each updated document one line so it won't take too much vertical space.

Reply:
added two more examples and formatted as one line. LMK if you have suggestions on formatting the ellipses.